connect/src/main/java/com/stripe/android/connect/StripeComponentView.kt

```diff
@@ -1,18 +1,259 @@
 package com.stripe.android.connect
 
+import android.app.Application
 import android.content.Context
+import android.content.res.ColorStateList
 import android.util.AttributeSet
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.WebView
 import android.widget.FrameLayout
+import android.widget.ProgressBar
 import androidx.annotation.RestrictTo
+import androidx.core.view.isVisible
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProvider.AndroidViewModelFactory.Companion.APPLICATION_KEY
+import androidx.lifecycle.findViewTreeLifecycleOwner
+import androidx.lifecycle.findViewTreeViewModelStoreOwner
+import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.viewmodel.MutableCreationExtras
+import com.stripe.android.connect.di.StripeConnectComponent
+import com.stripe.android.connect.util.AndroidClock
 import com.stripe.android.connect.webview.StripeConnectWebViewContainer
+import com.stripe.android.connect.webview.StripeConnectWebViewContainerState
+import com.stripe.android.connect.webview.StripeConnectWebViewContainerViewModel
+import com.stripe.android.core.Logger
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.launch
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.buildJsonObject
 
 @PrivateBetaConnectSDK
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-abstract class StripeComponentView<Listener, Props>(
+abstract class StripeComponentView<Listener, Props> internal constructor(
     context: Context,
     attrs: AttributeSet?,
     defStyleAttr: Int,
+    private val embeddedComponent: StripeEmbeddedComponent,
```

Comment:
👀 All the changes here are basically copied from `StripeConnectWebViewContainerImpl` with minor modification to replace `containerView` with `this` instance.